### PR TITLE
Grab the system architecture instead of assuming amd64

### PR DESCRIPTION
**Pull request by brendandburns**
This enables 'az' CLI + ACS on Raspberry PI (and others)

### CODE REVIEW

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 1]
```diff
@@ -278,14 +278,17 @@ def k8s_install_cli(client_version='latest', install_location=None):
 
     file_url = ''
     system = platform.system()
-    base_url = 'https://storage.googleapis.com/kubernetes-release/release/{}/bin/{}/amd64/{}'
+    base_url = 'https://storage.googleapis.com/kubernetes-release/release/v{}/bin/{}/{}/{}'
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: Think there's a bug here.
If `client_version` is `latest`, `https://storage.googleapis.com/kubernetes-release/release/stable.txt` returns a version that includes the `v` prefix so then `base_url` would have `/release/vv1.6.3`?

File: src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py [hunk 2]
```diff
@@ -278,14 +278,17 @@ def k8s_install_cli(client_version='latest', install_location=None):
...
+    arch = platform.machine().lower()
```
Comment (derekbekoe) [inline src/command_modules/azure-cli-acs/azure/cli/command_modules/acs/custom.py]: One my mac, `platform.machine()` returns `x86_64`.
`https://storage.googleapis.com/kubernetes-release/release/v1.6.3/bin/darwin/x86_64/kubectl` does not exist.

**Comment by troydai** [comment]
Ping @brendandburns

**Comment by tjprescott** [comment]
Ping @brendandburns

**Comment by troydai** [comment]
This pull request is closed since it hasn't been updated for months and concerns unaddressed. Please feel free to re-open this pull request in the future.

**Pull request closed without merging**